pegasus/migrations/105_populate_forms_hashed_email.rb

```diff
@@ -0,0 +1,30 @@
+require 'digest/md5'
+
+Sequel.migration do
```

Comment:
The migration should be merged and pushed to production before merging any PRs that rely on its existence, in case of rollback or migration failure.